### PR TITLE
CB-13081 DL upgrade on azure: package version check still throws

### PR DESCRIPTION
**Pull request by gergopapi2**
Previous CB-12684 disabled package version check during upgrade only in one part. There was another invocation of check package versions. This commit turns of checking package versions also for that other invocation

See detailed description in the commit message.

### CODE REVIEW

**Review by lnardai — APPROVED** [review]
LGTM! wait for one more approval from our team.

**Review by horadla23 — APPROVED** [review]

**Pull request merged**
